modules/packages/Sort.chpl

```diff
@@ -546,897 +546,1022 @@ iter sorted(x, comparator:?rec=defaultComparator) {
     yield i;
 }
 
-
-/* Sort Functions */
-
 /*
    Sort the 1D array `Data` in-place using a sequential bubble sort algorithm.
 
+   .. note:: This method is deprecated.
+
    :arg Data: The array to be sorted
    :type Data: [] `eltType`
    :arg comparator: :ref:`Comparator <comparators>` record that defines how the
       data is sorted.
 
  */
 proc bubbleSort(Data: [?Dom] ?eltType, comparator:?rec=defaultComparator) {
-  chpl_check_comparator(comparator, eltType);
-  const low = Dom.low,
-        high = Dom.high,
-        stride = abs(Dom.stride);
+  compilerWarning("bubbleSort is deperecated");
```

Comment:
typo: `deprecated`

Reply:
Would it be appropriate to redirect a user to the top-level `sort` function now? e.g.

```
"bubbleSort is deprecated - please use sort
```